services/jira/jira-issue.tester.js

```diff
@@ -162,26 +168,27 @@ t.create('blue-gray status color')
     value: 'cloudy',
     colorB: colorScheme.blue,
   })
-
-t.create('with auth')
-  .before(mockJiraCreds)
-  .get('/https/myprivatejira.com/secure-234.json')
-  .intercept(nock =>
-    nock('https://myprivatejira.com/rest/api/2/issue')
-      .get(`/${encodeURIComponent('secure-234')}`)
-      // This ensures that the expected credentials from serverSecrets are actually being sent with the HTTP request.
-      // Without this the request wouldn't match and the test would fail.
-      .basicAuth({
-        user,
-        pass,
-      })
-      .reply(200, {
-        fields: {
-          status: {
-            name: 'in progress',
+;[mockJiraCreds, mockLegacyJiraCreds].map(mockCreds => {
```

Comment:
Prettier doesn't want to let us have a blank line here :(

Reply:
Is it this issue? prettier/prettier#1510

Reply:
yeah - looks like it